dags/search_forecasting.py

```diff
@@ -0,0 +1,79 @@
+"""
+See [kpi-forecasting in the docker-etl repository](https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
+
+This DAG runs the search forecasts for the DAU, search count and ad clicks metrics .
+
+This DAG is high priority for week 1 of the month and low priority otherwise.
+"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskSensor
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.tags import Tag
+
+default_args = {
+    "owner": "jsnyder@mozilla.com",
+    "email": [
+        "jsnyder@mozilla.com",
+        "mbowerman@mozilla.com",
+        "telemetry-alerts@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 7, 6),
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+TAGS = [Tag.ImpactTier.tier_1]
+IMAGE = "gcr.io/moz-fx-data-airflow-prod-88e0/kpi-forecasting_docker_etl:latest"
+
+FORECAST_METRICS_LIST = [
+    "search_forecasting_daily_active_users",
+    "search_forecasting_search_count",
+    "search_forecasting_ad_clicks",
+]
+
+
+# all the search forecasting metrics come from the search_revenue_levers_daily
+# table which is run in the bqetl_search_dashboard dag
+# as the search_derived__search_revenue_levers_daily__v1 task
+# see: https://workflow.telemetry.mozilla.org/dags/bqetl_search_dashboard/grid
+wait_task_sensor = ExternalTaskSensor(
```

Comment:
This needs to be moved inside the DAG context manager.

Reply:
whoops, pushed a fix just now